src/pages/settings/AppDownloadLinks.js

```diff
@@ -70,4 +108,7 @@ const AppDownloadLinksPage = (props) => {
 AppDownloadLinksPage.propTypes = propTypes;
 AppDownloadLinksPage.displayName = 'AppDownloadLinksPage';
 
-export default withLocalize(AppDownloadLinksPage);
+export default compose(
```

Comment:
Can you please explain this change?

Reply:
Why should I change to this?

Reply:
I meant - Could you please explain me why you made this change

Reply:
I think since he is using more than HOCs he would have to pass them to compose one is withLocalize for translations and windowDimensions to respond to right-click or long press.

Reply:
Right!